python_modules/dagster/dagster/_core/pipes/client.py

```diff
@@ -77,6 +77,16 @@ def get_materialize_result(self) -> MaterializeResult:
         """
         return materialize_result_from_pipes_results(self.get_results())
 
+    def get_asset_check_result(self) -> AssetCheckResult:
+        """Get a single asset check result for a pipes invocation.
+
+        This does not work on invocations that have more than a single asset check result.
+        Use `get_results` instead to get the result stream in those cases.
+
+        Returns: AssetCheckResult
+        """
+        return _check_result_from_pipes_results(self.get_results())
+
```

Comment:
What is the behavior here is there is a single asset check attached to a single materialize result?